src/modules/launcher/Plugins/Microsoft.Plugin.Program/Storage/Win32ProgramRepositoryHelper.cs

```diff
@@ -0,0 +1,43 @@
+﻿using System;
+using System.Collections.Generic;
+using Wox.Infrastructure.Storage;
+
+namespace Microsoft.Plugin.Program.Storage
+{
+    internal class Win32ProgramRepositoryHelper
```

Comment:
nit:  I'm not a big fan of the term 'helper', what about Win32ProgramFileSystemWatchers?